boskos/client/client.go

```diff
@@ -150,6 +150,12 @@ func (c *Client) Reset(rtype string, state string, expire time.Duration, dest st
 	return c.reset(rtype, state, expire, dest)
 }
 
+// Metric will query current metric for target resource type.
+// Return a common.Metric object on success.
+func (c *Client) Metric(rtype string) (common.Metric, error) {
+	return c.metric(rtype)
```

Comment:
Just try to make it consistent here and hide the implementation from the public method.